lib/sky-pages-module-generator.js

```diff
@@ -52,19 +52,17 @@ function getSource(skyAppConfig) {
   ];
 
   if (skyAppConfig.runtime.includeRouteModule) {
-
     runtimeModuleImports.push('routing');
-
     runtimeProviders.push('appRoutingProviders');
+  } else {
+    /*
+      Import the regular RouterModule so that tested components can reference things
+      like routerLink
+    */
     runtimeProviders.push(`{
```

Comment:
I would probably suggest adding it a little higher up at the `runtimeProviders` declaration.